plugins/firebase/firebase/functions/index.ts

```diff
@@ -0,0 +1,292 @@
+import { NIL_UUID } from "@hot-updater/core";
+import { verifyJwtSignedUrl, withJwtSignedUrl } from "@hot-updater/js";
+import * as admin from "firebase-admin";
+import * as functions from "firebase-functions/v1";
+import { Hono } from "hono";
+import { cors } from "hono/cors";
+import { getUpdateInfo } from "./getUpdateInfo";
+
+declare global {
+  var HotUpdater: {
+    REGION: string;
+    BUCKET_NAME?: string;
+    JWT_SECRET: string;
+  };
+}
+
+if (typeof global.HotUpdater === "undefined") {
+  global.HotUpdater = {
+    REGION: process.env.FUNCTION_REGION || "us-central1",
+    BUCKET_NAME: process.env.STORAGE_BUCKET || undefined,
+    JWT_SECRET: process.env.JWT_SECRET || "your-secret-key",
+  };
+}
+
+if (!admin.apps.length) {
+  admin.initializeApp();
+}
+
+export function validatePlatform(platform: string): "ios" | "android" | null {
```

Comment:
Don’t set default values arbitrarily. Instead of assigning defaults, throw an error so the endpoint fails explicitly. It’s often worse to let something pass silently due to an inappropriate default value.